libsolidity/ast/Types.cpp

```diff
@@ -3090,6 +3090,14 @@ string FunctionType::canonicalName() const
 	return "function";
 }
 
+string FunctionType::humanReadableName() const
+{
+	switch (m_kind) {
+		case Kind::Error: return "error";
```

Comment:
```suggestion
		case Kind::Error: return "error " + m_declaration->name();
```
We could add the name of the error as requested by @cameel  in https://github.com/ethereum/solidity/issues/12896#issuecomment-1108853261